pkg/openstack/galera.go

```diff
@@ -49,6 +49,14 @@ func ReconcileGaleras(
 			IssuerName: instance.GetInternalIssuer(),
 			CertName:   fmt.Sprintf("galera-%s-svc", name),
 			Hostnames:  []string{hostname},
+			// Note (dciabrin) from https://github.com/openstack-k8s-operators/openstack-operator/pull/678#issuecomment-1952459166
+			// the certificate created for galera should populate the 'organization' field,
+			// otherwise this trip the SST transfer setup done by wsrep_sst_rsync. This will not show
+			// at the initial deployment because there is no SST involved when the DB is bootstrapped
+			// as there are no data to be transferred yet.
+			Subject: &certmgrv1.X509Subject{
+				Organizations: []string{"cluster.local"},
```

Comment:
was thinking `<namespace>.cluster.local` for the org in the QEMU certs, probably doesn't matter to much though

Reply:
updated it to that. could add a func to the tls certmanager module to return a default org for this?